apps/web/pages/reschedule/[uid].tsx

```diff
@@ -125,20 +131,23 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   const eventType = booking.eventType ? booking.eventType : getDefaultEvent(dynamicEventSlugRef);
 
-  const eventPage = `${
-    eventType.team
-      ? `team/${eventType.team.slug}`
-      : dynamicEventSlugRef
-      ? booking.dynamicGroupSlugRef
-      : booking.user?.username || "rick" /* This shouldn't happen */
-  }/${eventType?.slug}`;
+  const enrichedBookingUser = booking.user
+    ? await UserRepository.enrichUserWithItsProfile({ user: booking.user })
+    : null;
+
+  const eventPage = await buildEventUrlFromBooking({
+    eventType,
+    dynamicGroupSlugRef: booking.dynamicGroupSlugRef ?? null,
```

Comment:
Using just `dynamicGroupSlugRef` instead of checking for `dynamicEventSlugRef` nullish status and then using `dynamicGroupSlugRef`.


`dynamicGroupSlugRef` and `dynamicEventSlugRef` are set together. So, I don't see a different practically,